src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```diff
@@ -391,6 +398,8 @@ private NuGetPackageInfo ReadPackageInformation(string packageLocation)
 
             return new NuGetPackageInfo(
                 nuspec.GetAuthors(),
+                nuspec.GetOwners(),
+                PrefixReserved,
```

Comment:
nit: we may just return `false` here, property on `NuGetInstaller` level is a bit confusing.